globalconfig.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # Caution: This file is read by vagrant, ansible and bash
 vm_name: "Quantum Mobile"
-vm_version: "20.05.0"
+vm_version: "20.06.0"
```

Comment:
@ltalirz just want to clarify the protocol here for version naming: `<year>.<major>.<minor>`?

Reply:
I think it's `<year>.<month>.<minor>`, right @ltalirz ?

Reply:
yes